src/command.c

```diff
@@ -524,6 +535,176 @@ int cmd_quit(struct command *cmd)
     return CORVUS_OK;
 }
 
+static int cmd_parse_len(struct redis_data *data, int *result)
+{
+    ASSERT_TYPE(data, REP_STRING);
+    char len_limit[data->pos.str_len + 1];
+    if (pos_to_str(&data->pos, len_limit) == CORVUS_ERR) {
+        LOG(ERROR, "parse_len: emptry arg string");
+        return CORVUS_ERR;
+    }
+
+    int v = 0;
+    for (size_t i = 0; i != data->pos.str_len; i++) {
+        char c = len_limit[i];
+        if (c < '0' || c > '9') {
+            LOG(ERROR, "parse_len: char not between 0-9");
+            return CORVUS_ERR;
+        }
+        v = 10 * v + (c - '0');
+    }
+    *result = v;
+
+    return CORVUS_OK;
+}
+
+int cmd_slowlog_get(struct command *cmd, struct redis_data *data)
+{
+    // For example 'slowlog get 128', element[2] is 128 here
+    int len = config.slowlog_max_len;
+    if (data->elements > 3) {
+        LOG(DEBUG, "cmd_slowlog_get: too many arguments");
+        return CORVUS_ERR;
+    } else if (data->elements == 3) {
+        int len_limit;
+        struct redis_data *len_limit_data = &data->element[2];
+        if (cmd_parse_len(len_limit_data, &len_limit) == CORVUS_ERR) {
+            return CORVUS_ERR;
+        }
+        if (len_limit == 0) {
+            LOG(ERROR, "cmd_slowlog_get: invalid len");
+            return CORVUS_ERR;
+        }
+        if (len_limit < len) {
+            len = len_limit;
+        }
+    }
```

Comment:
`data->elements < 3` means only `slowlog get` is sent, in this case `len` is setted to `config.slowlog_max_len`(line 564), the default slowlog queue max length.